navigator/frontend/yarn.lock

```diff
@@ -4944,11 +4944,16 @@ modernizr@^3.11.8:
     requirejs "^2.3.6"
     yargs "^15.4.1"
 
-moment@2.29.1, moment@^2.22.1, moment@^2.23.0:
+moment@2.29.1:
```

Comment:
It doesn't look like this actually removes transitive dependencies on the library. I'll create a PR myself.